src/Nelmio/Alice/Fixtures/Builder/Builder.php

```diff
@@ -26,6 +27,11 @@ class Builder
      */
     protected $templates;
 
+    /**
+     * Builder constructor.
```

Comment:
I agree for the `@param`, but not for the comment: it doesn't add any value here so just end up creating some noise